composer.json

```diff
@@ -19,8 +19,8 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
-        "composer/xdebug-handler": "^3.0",
```

Comment:
To make things work, I had to change some packages' versions. I have tested that they are still working as expected.